packages/eddsa-poseidon/src/utils.ts

```diff
@@ -1,5 +1,5 @@
 import { Point } from "@zk-kit/baby-jubjub"
-import type { BigNumberish } from "@zk-kit/utils"
+import { type BigNumberish } from "@zk-kit/utils"
```

Comment:
I honestly don't know what this `type` keyword is for.  The build works fine for me locally if I simply remove it entirely, so I suggest doing that if it passes the GitHub checks.

Reply:
Imports could be values or types. TS can infer what kind of import that is but the `type` keyword here makes it more explicit.